numpy/lib/_npyio_impl.py

```diff
@@ -561,6 +563,10 @@ def save(file, arr, allow_pickle=True, fix_imports=True):
     >>> print(a, b)
     # [1 2] [1 3]
     """
+    if fix_imports is not None:
+        warnings.warn(
```

Comment:
We try to always add a comment like:
```
        # Deprecated 2024-05-16, NumPy 2.1
```
and also add `(Deprecated in NumPy 2.1)` at the end of the warning.  Or 2.0 if we backport.